README.md

```diff
@@ -60,6 +60,16 @@ eval_output = eval_algo.evaluate(model=model_runner, dataset_config=config)
 [examples](https://github.com/aws/fmeval/tree/main/examples) for more details around the usage of
 eval algorithms.*
 
+## Telemetry
+By default, `fmeval` has very basic telemetry enabled, purely for tracking usage of the package.
```

Comment:
Let's rephrase this to below:
```
`fmeval` has a telemetry enabled, for tacking the usage of AWS provided/hosted LLMs.
This data is tracked using the number of SageMaker, JumpStart, or Bedrock `ModelRunner` objects that get created. 
This can be disabled by setting the `DISABLE_FMEVAL_TELEMETRY` as `false`.
```

Question:
* Are we doing this for Bedrock? If not, then let's remove it.

Reply:
We're not doing this for Bedrock; that was my mistake. Will correct this alongside the rest of the README edits.